web/src/components/snapshots/Snapshots.jsx

```diff
@@ -108,12 +108,8 @@ class Snapshots extends Component {
       }
       const response = await res.json();
 
-      const snapshots = response.backups?.flatMap(
-        (replBackup) => replBackup?.backups
-      );
-
       this.setState({
-        snapshots: snapshots?.sort((a, b) =>
+        snapshots: response.backups?.sort((a, b) =>
```

Comment:
This just reverts what we've done in - https://github.com/replicatedhq/kots/pull/5033/files#diff-84ad0b83d6595420183ed4f5d5f3b47be290c7a5a014f1cc4729329c2ac3dc2a

The main difference we introduced has been 2 extra fields (the backup count and expected) and a reduction of potential status phases:
- https://github.com/replicatedhq/kots/blob/05573a551f67706e5fa91fc29e271c4188cc497f/pkg/kotsadmsnapshot/types/types.go#L66-L75

I've checked in:
- https://github.com/replicatedhq/kots/blob/5fbcb34f8a5ba4a2999c2bb02e510138b115cf9b/web/src/components/snapshots/SnapshotRow.jsx#L36-L142

And it doesn't look like we've introduced anything different or non standard to what we're already using ☝️  CC @miaawong if you could confirm 👀